zuul/connection/github.py

```diff
@@ -185,6 +185,20 @@ def _event_pull_request_review(self, request):
         event.type = 'pr-review'
         return event
 
+    def _event_status(self, request):
+        body = request.json_body
+        action = body.get('action')
+        if action == 'pending':
+            return
```

Comment:
Seems like a hard decision here to not allow triggering on a state of `pending`. But we report up the state of either `success`. Shouldn't we also report `pending` and let layout authors decide if they want to do anything on pending or not?